dataland-frontend/tests/e2e/fixtures/lksg/LksgDataFixtures.ts

```diff
@@ -47,46 +48,26 @@ export function generateOneLksgFixtureWithManyNulls(): FixtureData<LksgData> {
   )[0];
 }
 
```

Comment:
Is there a good reason for these numerous changes to the Lksg Data Fixtures? Just cleaning up code? Addressing items from the refactoring log?